### DISCUSSION TITLE
DAI 2023-2024 - Practical work 2 - Lopez Esteban and Bleuer Rémy

### DISCUSSION THREAD

**Discussion by RemyBlr** [heig-vd-dai-course/heig-vd-dai-course#138] [Show and tell]
Live chat between one or multiple clients connected to the server. 
In first place the user choses to chat with one person or all the people connected to the server.
If we have time, we were thinking about the possibility to create groups and write messages to this group only

**Comment by EstebanLopezGomez** [comment]
Hello @ludelafo @hadrylouis, here's the hash for the last commit of our Practical Work 2 : 76a5e3bb30d9bf895637f3dd8cc0966e6b9f1ad2
Hope you like what we have done!

Cordially 

Esteban Lopez & Rémy Bleuer

**Reply by EstebanLopezGomez** [reply]
Hi again, here's the link to the repository : [https://github.com/RemyBlr/dai-bleuer-lopez-practical-work-2](https://github.com/RemyBlr/dai-bleuer-lopez-practical-work-2) 

sorry about the inconvenience and have a great day.

Cordially

Esteban Lopez & Rémy Bleuer
